tests/update/patch_before_migration.pm

```diff
@@ -52,11 +55,15 @@ sub patching_sle {
 
     if (get_var('FULL_UPDATE')) {
         fully_patch_system();
-        type_string "reboot\n";
-        $self->wait_boot(textmode => !is_desktop_installed(), ready_time => 600, bootloader_time => 200);
-
-        # Go back to the initial state, before the patching
-        $self->setup_migration();
+        # Update origin system on zVM that is controlled by autoyast profile and reboot is done by end of autoyast installation
+        # So we skip reboot here after fully patched on zVM to reduce times of reconnection to s390x
+        if (!get_var('UPGRADE_ON_zVM')) {
```

Comment:
Please use only uppercase for test variables. I think we do not have mixed lower+upper elsewhere.

Reply:
Fixed.